pulseapi/entries/views.py

```diff
@@ -187,11 +186,12 @@ class EntryView(RetrieveAPIView):
 
 
 class BookmarkedEntries(ListAPIView):
+    pagination_class = EntriesPagination
 
     def get_queryset(self):
         user = self.request.user
         bookmarks = UserBookmarks.objects.filter(user=user)
-        return Entry.objects.filter(bookmarked_by__in=bookmarks)
+        return Entry.objects.filter(bookmarked_by__in=bookmarks).order_by('-bookmark_by')
```

Comment:
It was done by trial and error so im actually not 100% sure if i did it right. Basically I want the returned entries to be sorted by recently fav’ed to least recently.

I thought `.order_by('-bookmark_by')` is to order the entries by their ids appear in the `users_userbookmarks` table?